test/system/test_tpm2_pcrextend.sh

```diff
@@ -45,6 +45,7 @@ onerror() {
 }
 trap onerror ERR
 
+digests=""
 # test a single algorithm based on what is supported
 for alg in `tpm2_pcrlist -s | cut -d\  -f 3-`; do
```

Comment:
Maybe in a different patch at some date, we can use test_helpers.sh that can be sourced. Looking at this maybe using hash_alg_supported function would be better.